src/commands/createAndSendCommand.ts

```diff
@@ -39,11 +39,16 @@ export const createAndSendTxCommand = {
         describe: "activates sudo mode",
         type: "boolean",
         default: false,
+      })
+      .positional("immortality", {
```

Comment:
BTW, any reason why all the parameters are positional ? I always find it less flexible that way

Reply:
As opposed to making them optional you mean?

Reply:
@crystalin ?

Reply:
please point me to the right feature because I'm not aware of any other way to do that